src/__experimental__/components/date/date.spec.js

```diff
@@ -480,6 +480,16 @@ describe('Date', () => {
       });
     });
 
+    it('short circuits when either "disabled" and/or "readOnly"', () => {
+      wrapper = render({});
+      wrapper.find(InputIconToggle).props().onClick();
+      wrapper.setProps({ disabled: true, readOnly: true });
```

Comment:
It would seem that calling .setProps is necessary here, simply set these props in render like so `wrapper = render({ disabled: true, readOnly: true });

Reply:
This was to address an issue I noticed in Storybook, where you can focus the input and then set a `disabled` or `readOnly` knob and still interact with the picker after. The props need to be set in this way to test the scenario.

Reply:
Ok that makes sense then :)